spec/services/active_department_employees_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ActiveDepartmentEmployeesService do
+  context 'when there are no records' do
+    let(:result) { described_class.new.call }
+
+    it 'returns an empty array' do
+      expect(result).to be_empty
+    end
+  end
+
+  context 'when there are saved records' do
```

Comment:
```suggestion
  context 'when there are active employees' do
```
Remeber to me more specific with contexts.